extensions/expression-language/base64.h

```diff
@@ -0,0 +1,188 @@
+/***************************************************************************
```

Comment:
Why is this class here?

Reply:
This is the cURL base64 encode/decode. Here's why:

1) Avoid reinventing the wheel
2) It's a private function in cURL, so that's why it's copied rather than exposing/calling into cURL lib
3) We already have cURL included & in LICENSE, so it's simple to use this one
4) Simpler implementation to call than OpenSSL implementation

Reply:
Rebased to fix conflicts. The reason it's here rather than 3rd-party is that it's specific functions copied/pasted, not a third party lib. We own/adopted the code now and it won't be maintained by a third party.

Reply:
Thanks for rebasing I will merge.